cloud_run_healthchecks_startup_probe_gRPC/main.tf

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2022 Google LLC
```

Comment:
When I run this one, I get:

```
 Error: Error waiting to create Service: resource is in failed state "Ready:False", message: Revision 'cloudrun-service-healthcheck-t6z2v' is not ready and cannot serve traffic. The user-provided container failed the configured startup probe checks. Logs for this revision might contain more information.
│
│ Logs URL: https://console.cloud.google.com/logs/viewer?project=subtle-abacus-220417&resource=cloud_run_revision/service_name/cloudrun-service-healthcheck/revision_name/cloudrun-service-healthcheck-t6z2v&advancedFilter=resource.type%3D%22cloud_run_revision%22%0Aresource.labels.service_name%3D%22cloudrun-service-healthcheck%22%0Aresource.labels.revision_name%3D%22cloudrun-service-healthcheck-t6z2v%22
│ For more troubleshooting guidance, see https://cloud.google.com/run/docs/troubleshooting#container-failed-to-start
│
│   with google_cloud_run_service.default,
│   on main.tf line 13, in resource "google_cloud_run_service" "default":
│   13: resource "google_cloud_run_service" "default" {
```

Maybe this can be fixed with `depends_on`?

Reply:
Or see:

https://githubhelp.com/google/exposure-notifications-verification-server